qt/scientific_interfaces/Indirect/Iqt.cpp

```diff
@@ -31,14 +34,71 @@ std::size_t getWsNumberOfSpectra(std::string const &workspaceName) {
   return getADSMatrixWorkspace(workspaceName)->getNumberHistograms();
 }
 
-bool checkADSForWorkspace(std::string const &workspaceName) {
+bool doesExistInADS(std::string const &workspaceName) {
```

Comment:
Do we  really need this function? Can we not just call ADS.doesExist when we need it?